pymc_marketing/mmm/delayed_saturated_mmm.py

```diff
@@ -23,31 +24,48 @@
 import numpy.typing as npt
 import pandas as pd
 import pymc as pm
+import pytensor.tensor as pt
 import seaborn as sns
 from pytensor.tensor import TensorVariable
 from xarray import DataArray, Dataset
 
 from pymc_marketing.constants import DAYS_IN_YEAR
-from pymc_marketing.mmm.base import MMM
+from pymc_marketing.mmm.base import BaseValidateMMM
+from pymc_marketing.mmm.budget_optimizer import BudgetOptimizer
+from pymc_marketing.mmm.components.adstock import (
+    AdstockTransformation,
+    _get_adstock_function,
+)
+from pymc_marketing.mmm.components.saturation import (
+    SaturationTransformation,
+    _get_saturation_function,
+)
 from pymc_marketing.mmm.lift_test import (
-    add_logistic_empirical_lift_measurements_to_likelihood,
+    add_lift_measurements_to_likelihood,
     scale_lift_measurements,
 )
-from pymc_marketing.mmm.preprocessing import MaxAbsScaleChannels, MaxAbsScaleTarget
-from pymc_marketing.mmm.transformers import geometric_adstock, logistic_saturation
+from pymc_marketing.mmm.preprocessing import (
+    MaxAbsScaleChannels,
+    MaxAbsScaleTarget,
+)
 from pymc_marketing.mmm.tvp import create_time_varying_intercept, infer_time_index
 from pymc_marketing.mmm.utils import (
+    _get_distribution_from_dict,
     apply_sklearn_transformer_across_dim,
     create_new_spend_data,
     generate_fourier_modes,
 )
-from pymc_marketing.mmm.validating import ValidateControlColumns
+from pymc_marketing.mmm.validating import (
+    ValidateControlColumns,
+)
 
-__all__ = ["DelayedSaturatedMMM"]
+__all__ = ["BaseMMM", "MMM", "DelayedSaturatedMMM"]
 
 
-class BaseDelayedSaturatedMMM(MMM):
-    """Base class for a media mix model with delayed adstock and logistic saturation class (see [1]_).
+class BaseMMM(BaseValidateMMM):
+    """
+    Base class for a media mix model using Delayed Adstock and Logistic Saturation,
+    as default initialization (see [1]_).
```

Comment:
There are no defaults for adstock and saturation and think that is fine to keep so think this is inaccurate

Reply:
Done